todoapp/codelab/25mar2017/01_todo_content.md

```diff
@@ -0,0 +1,44 @@
+
+Bienvenido a este codelab de location. Este code lab cuenta con 5 pasos siendo este el primero. Vamos a empezar actualizando nuestro Modelo ToDo.
+
+#ToDoContent
+
+
+En `ToDoContent.java` vamos a agregar 2 propiedades nuevas latitude y longitud con estos datos podemos contruir un Location. Tambien agregaremos un nuevo contructor que acepte estes datos y sus respectivos getter y setter.
+```java
+public class ToDoContent implements Serializable {
+..
+private double lat;
+private double lng;
+..
+  public ToDoContent(String title, Date date, boolean hasDate, double lat, double lng) {
+        this.title = title;
+        this.date = date;
+        this.hasDate = hasDate;
+        this.lat = lat;
+        this.lng = lng;
+    }
+..
+public double getLat() {
+        return lat;
+    }
+
+    public void setLat(double lat) {
+        this.lat = lat;
+    }
+
+    public double getLng() {
+        return lng;
+    }
+
+    public void setLng(double lng) {
+        this.lng = lng;
+    }
+}
+```
+
+Ahora ya podemos guardar latitude y longitud en nuestro ToDo.
```

Comment:
Please explain what is lat and lng if possible, give a brief explanation WHY are we going to store them, it is not just "add code and that's it!"